### PR TITLE
use the new installer certs to serve

### PR DESCRIPTION
**Pull request by deads2k**
switch to the specific kube-apiserver serving certs for serving the API.

### CODE REVIEW

File: bindata/bootkube/config/config-overrides.yaml [hunk 1]
```diff
@@ -6,9 +6,6 @@ kubeletClientInfo:
   keyFile: /var/run/secrets/kubelet-client/tls.key
 serviceAccountPublicKeyFiles:
 - /var/run/configmaps/sa-token-signing-certs/ca-bundle.crt
-servingInfo:
```
Comment (sttts) [inline bindata/bootkube/config/config-overrides.yaml]: where are these set now? in defaultconfig.yml?

Comment (deads2k) [reply bindata/bootkube/config/config-overrides.yaml]: > where are these set now? in defaultconfig.yml?

the bootstrap has another override file that already has these.

**Comment by deads2k** [comment]
@abhinavdahiya @sdodson what's the ETA for bootstrap container logs and master container logs on failing installs?

**Comment by abhinavdahiya** [comment]
This PR has not gone green even once. This does not look like a flake.

Can you hold and see if the actually works? Because otherwise the bot is going to keep retesting.


Also I didn't see any exception for this.

**Comment by sdodson** [comment]
/hold
Clear this after we believe the tests will pass.

**Comment by deads2k** [comment]
Hmmm... One node failing?

/retest

**Comment by deads2k** [comment]
@derekwaynecarr this is borked, but without bootstrap logs I can't figure out why the operators are failing. 
 We need this pull (well, one nearly identical), to land the fixes for the kubelet/kube-apiserver trust.

**Pull request merged**